account_reporting_weight/__manifest__.py

```diff
@@ -1,9 +1,9 @@
-# Copyright 2017 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# Copyright 2017 Tecnativa - Pedro M. Baeza
```

Comment:
Copyright 2012

Reply:
I do not understand what you are suggesting, only the email is deleted to avoid spam.

Reply:
The year... but it is ok